### PR TITLE
corrected candidate table and some bugs

### PR DESCRIPTION
**Pull request by SaranchukVadym**

### CODE REVIEW

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+databaseChangeLog:
+  - changeSet:
+      id: run-fix-table-candidate
+      author: Saranchuk Vadim
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml:4]: please add precondition

File: sandbox-backend/src/main/resources/db/changelog/db.changelog-master.yaml [hunk 2]
```diff
@@ -58,8 +58,9 @@ databaseChangeLog:
   - include:
       file: db/changelog/changes/20210413_add_column_emp_password.yaml
   - include:
+      file: db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/db.changelog-master.yaml:61]: need new -include for all file, it do not work with one include and several files

Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/db.changelog-master.yaml:61]: corrected

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml (hunk 1 above)
Comment (SaranchukVadym) [reply sandbox-backend/src/main/resources/db/changelog/changes/2021.04.17_run_fix_table_candidate.yaml:4]: corrected

**Pull request closed without merging**